src/main/java/com/google/api/generator/engine/escaper/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "escaper_files",
+    srcs = glob(["*.java"]),
+)
+
+java_library(
+    name = "escaper",
+    srcs = [
+        ":escaper_files",
+    ],
+    deps = [
+        "@com_google_guava_guava//jar"
```

Comment:
Is this dependency used in this packageÉ

Reply:
this is no longer needed until we add comment escaper here. Thanks!